internal/cloud/realm/error.go

```diff
@@ -3,13 +3,31 @@ package realm
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
 	"net/http"
 )
 
 const (
 	invalidSessionCode = "InvalidSession"
 )
 
+// ErrInvalidSession is an invalid session error that has a follow up message
+type ErrInvalidSession struct {
+	error
```

Comment:
we can get rid of this `error` embed, and the constructor for this type.  instead, declare/return it with just `realm.ErrInvalidSession{}`

what you can do here for now is add a field `profileName string` to this struct definition.  for now, don't worry about setting that field's value whenever you declare this error (that'll be for another ticket).  but we can use/reference that field in the `SuggestedCommands` implementation below

you will also need to implement `func (err ErrInvalidSession) Error() string { return "invalid session" }` here